Bottom: aecf4dc940f0dd6212374b753e38c97002489a1a
Top:    bf07d3b8c8c2e488b66bc951e75adc947f82dc0f
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 17:14:13 -0700

Refresh of iohandler-fall-back-on-file

---
diff --git a/lldb/source/Core/IOHandler.cpp b/lldb/source/Core/IOHandler.cpp
index e82c91da16a..a5e28d1a640 100644
--- a/lldb/source/Core/IOHandler.cpp
+++ b/lldb/source/Core/IOHandler.cpp
@@ -355,7 +355,7 @@ bool IOHandlerEditline::GetLine(std::string &line, bool &interrupted) {
 
       while (!done) {
         size_t bytes_read = sizeof(buffer);
-        m_input_sp->GetFile().Read((void *)buffer, bytes_read);
+        m_input_sp->Read((void *)buffer, bytes_read);
         if (bytes_read) {
           auto bytes = llvm::StringRef(buffer, bytes_read);
           size_t pos = bytes.find('\n');
